darts/ad/utils.py

```diff
@@ -422,105 +425,259 @@ def show_anomalies_from_scores(
 
         nbr_plots = nbr_plots + len(set(window))
 
-    fig, axs = plt.subplots(
-        nbr_plots,
-        figsize=(8, 4 + 2 * (nbr_plots - 1)),
-        sharex=True,
-        gridspec_kw={"height_ratios": [2] + [1] * (nbr_plots - 1)},
-        squeeze=False,
-    )
+    if multivariate_plot:
```

Comment:
I think that it would be great to refactor the logic to make it more concise and avoid code duplication; you could define an helper function with the "old" logic, potentially adding some features (for the axis/title naming for example) that `show_anomalies_from_scores()` would call in slightly different ways depending on the value of `multivariate_plot`.